src/main/java/uk/gov/ch/repository/officers/ActiveOfficersDetailsRepository.java

```diff
@@ -14,6 +14,13 @@
             + "cba.officer_forename_2 AS fore_name_2, "
             + "cba.officer_surname AS surname, "
             + "cba.occupation_desc AS occupation, "
+            + "apt.appointment_type_desc as role, "
```

Comment:
we should have the queries separate in different repositories as we don't want to modify the directors query as this is working and has been tested. yBy keeping them separate we can then delete the query , controller and model for the endpoint that is not needed when we switch to 5 officers. This will mean creating a new model class probably with a similar structure to return the officer query results.